src/components/mote/components/Motelandingsside.tsx

```diff
@@ -86,20 +94,47 @@ export const MotelandingssideMoteElement = (
 
 interface MotelandingssideProps {
   fnr: string;
-  mote: MoteDTO;
-  skalViseMotePrikk: boolean;
 }
 
 export const Motelandingsside = (
   motelandingssideProps: MotelandingssideProps
 ) => {
-  const { fnr, mote, skalViseMotePrikk } = motelandingssideProps;
+  const { fnr } = motelandingssideProps;
+  const tilgangState = useAppSelector((state) => state.tilgang);
+  const moterState = useAppSelector((state) => state.moter);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(hentMoter(fnr));
+  }, [fnr]);
+
+  const henter = tilgangState.henter || moterState.henter;
+  const hentingFeilet = tilgangState.hentingFeilet;
+  const tilgang = tilgangState.data;
+  const aktivtMote = moterState.data.find((mote) => mote.status !== "AVBRUTT");
+  const skalViseMotePrikk = isUnfinishedMoterTask(moterState);
+
+  if (henter) {
+    return <AppSpinner />;
```

Comment:
En alternativ vri på spinner, slik at man slipper if-sjekker for å bestemme hva man skal vise frem: https://github.com/navikt/syfooversikt/blob/master/src/components/AppSpinner.tsx

..samme pattern kan man også bruke på Feilmelding-komponenten.

Smak og behag, men jeg synes det blir litt mer clean

Reply:
Nice, takk for tips! Kan se på å bytte de ut i en ny PR.